app/assets/stylesheets/common/base/sidebar-section.scss

```diff
@@ -31,12 +29,26 @@
   .sidebar-section-header {
     @include ellipsis;
     flex: 1 1 auto;
+    text-transform: uppercase;
+    font-weight: bold;
     font-size: var(--font-1);
     color: var(--primary);
     align-items: center;
 
     &.sidebar-section-header-collapsable {
-      cursor: pointer;
+      background: transparent;
```

Comment:
Not sure is that would work in this case, but for buttons almost without styling, the `btn-flat` class can be  used:

https://github.com/discourse/discourse/blob/7b51ac418b669fe1ff8ecead6a211a4fc861f101/app/assets/stylesheets/common/components/buttons.scss#L261-L275